### ISSUE TITLE
Softmax Cross-Entropy Loss Function

### ISSUE THREAD

**Issue by prijall** [Open-Deep-ML/DML-OpenProblem#451]
It’s about stable softmax which is just a numerically safer way to compute the softmax function and it is used inside the cross-entropy loss computation which can be very useful to learn as beginner!!
